module/Application/src/Controller/IndexController.php

```diff
@@ -20,6 +20,11 @@ public function langAction(): Response
             $session->lang = 'nl';
         }
 
+        $url = $this->getRequest()->getQuery()['href'];
+        if (isset($url)) {
```

Comment:
If `href` is not a key this will still throw an error. Better would be `->getQuery('href')` and then checking `null` (I think).

And a question, where does `href` come from? I don't see it in the code.